decidim-admin/app/commands/decidim/admin/create_participatory_process.rb

```diff
@@ -65,6 +66,11 @@ def create_participatory_process
           process
         end
       end
+
+      def participatory_process_group
+        return unless form.participatory_process_group_id == 0
```

Comment:
What if you want to remove the PP from the PPG? I'd use where.first instead of find